src/index.html

```diff
@@ -5,8 +5,451 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img 
+            class="header__img" 
+            src="/images/logo.png" 
+            alt="mojo-logo"
+          >
+        </a>
+      </div>      
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link">
+              gadgets            
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav> 
+    </header>
+
+    <main class="page">
+      <div class="container page__container">
+        <section class="card container__card" data-qa="card">        
+          <img 
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="photo of an imac"
+          > 
+        
+          <div class="card__product-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__sku">
+            Product code: 195434 
+          </div>
+        
+          <div class="review card__review">
+            <div class="star stars--4">
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+            </div>
+            <text class="review__text">
+              Reviews: 5
+            </text>
+          </div>
+
+          <div class="card__price">
```

Comment:
you should have kept that `price` block and implement this approach
this is whole sheet
https://mate-academy.github.io/fe-program/css/typical-bem-mistakes.html
this part to refer
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/100297341/184828807-ab3c36f0-e846-497d-b6d0-9bb0395894f1.png">

Reply:
changed. thanks.